CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.0.0-dev.2.2
+
+* Added `UNUSED_OBJECT` hint (#80).
```

Comment:
This is great, thanks!  Is there a unit test for this?

I did end up making my own test object, which works well:  [UnusedObjects.zip](https://github.com/KhronosGroup/glTF-Validator/files/2048279/UnusedObjects.zip)